.github/workflows/ci-cleanup-old-workflow-runs.yml

```diff
@@ -0,0 +1,27 @@
+---
+# Copyright 2022 The Magma Authors.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: "Cleanup Workflows"
+
+on:
+  schedule:
+    - cron: '42 * * * *'
+
+jobs:
+  cleanup-workflows:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: Mattraks/delete-workflow-runs@c1a2f0de3fa4134a2357de83c92acff6e298bb8a # pin@v2.0.3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
According to the [documentation](https://github.com/Mattraks/delete-workflow-runs#2-repository), `repository` is also a required input. Or is there a reason it isn't required here?

Reply:
Yes, is required, however [it has a default](https://github.com/Mattraks/delete-workflow-runs#default--githubrepository-), which is fine for us.

Reply:
Oops, I somehow overlooked this.